### PR TITLE
added Splunk flow to standard artifact definitions

### PR DESCRIPTION
**Pull request by DeemOnSecurity**
Created a Splunk artifact flow with proper parameters and instructions (as there are a few more steps than Elastic).

I have tested this change with the builtin hunts and this pull should not be breaking.

### CODE REVIEW

File: artifacts/definitions/Splunk/Upload.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                                    timestamp(epoch=now()) AS timestamp,
+                                    ClientId, Flow.session_id AS FlowId,
+                                    "artifact_" + regex_replace(source=_value,
+                                       re='[/.]', replace='_') as _index
+                             FROM source(
+                                client_id=ClientId,
+                                flow_id=Flow.session_id,
+                                artifact=_value)
+                         })
+                         WHERE log(message=_value)
```
Comment (scudette) [inline artifacts/definitions/Splunk/Upload.yaml:72]: This log message basically duplicates the message above right? Maybe we can drop one of them?

Comment (DeemOnSecurity) [reply artifacts/definitions/Splunk/Upload.yaml:72]: This bit of code was copied from the Elastic Flow so we were achieving parity between the two. I can test removing it, but we may want to look into removing the same bit for the Elastic Flow!

Comment (scudette) [reply artifacts/definitions/Splunk/Upload.yaml]: The Elastic flow has no logs here

https://github.com/Velocidex/velociraptor/blob/43daec752774104a449eca99f8a0df6605f25a95/artifacts/definitions/Elastic/Flows/Upload.yaml#L42

The logs are useful for debugging but could get a bit much for production?

Comment (DeemOnSecurity) [reply artifacts/definitions/Splunk/Upload.yaml]: Interesting. Perhaps that was the code you dropped in the Discord which you were likely working on 😅. I have removed the redundant log line.

**Pull request merged**